include/cantera/kinetics/Kinetics.h

```diff
@@ -1203,7 +1203,7 @@ class Kinetics
      * @param i   reaction index
      */
     virtual bool isReversible(size_t i) {
-        throw NotImplementedError("Kinetics::isReversible");
+        return std::find(m_revindex.begin(), m_revindex.end(), i) < m_revindex.end();
```

Comment:
I believe this won't need to be virtual any longer, correct?

Reply:
Correct, and it should be `const` as well.